firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerformanceInitializer.java

```diff
@@ -15,17 +15,26 @@
 package com.google.firebase.perf;
 
 import com.google.firebase.perf.application.AppStateMonitor;
+import com.google.firebase.perf.logging.AndroidLogger;
 
 /**
  * FirebasePerformanceInitializer to initialize FirebasePerformance during app cold start
  *
  * @hide
  */
 public final class FirebasePerformanceInitializer implements AppStateMonitor.AppColdStartCallback {
+  private static final AndroidLogger logger = AndroidLogger.getInstance();
 
   @Override
   public void onAppColdStart() {
     // Initialize FirebasePerformance when app cold starts.
-    FirebasePerformance.getInstance();
+    try {
+      FirebasePerformance.getInstance();
+    } catch (IllegalStateException ex) {
+      logger.info(
```

Comment:
Should we elevate the log level to `Warning`?

Reply:
Made them all log as warnings